lib/editor/components/ExceptionDate.js

```diff
@@ -104,68 +102,64 @@ export default class ExceptionDate extends Component<Props, State> {
-          justifyContent: 'center',
-          textAlign: 'center',
-          position: 'relative',
-          marginBottom: '5px'
-        }}>
+      <FormGroup validationState={validationState}>
         <div
-          style={{gridColumn: '1 / 6'}}
-        >
-          <DateTimeField
-            inputProps={{style: {...inputStyleProps, backgroundColor: isInvalid ? '#FBDEDE' : undefined}}}
-            key={`date-${index}`}
-            mode='date'
-            {...dateTimeProps}
-          />
-        </div>
-
-        <Button
-          data-test-id='exception-add-range'
-          disabled={!date}
-          onClick={this._addRange}
-          size='sm'
-          style={{
-            height: '100%',
-            textAlign: 'center',
-            gridColumn: '7 / 12',
-            lineHeight: 0,
-            fontSize: 'x-small',
-            padding: '4',
-            margin: 0,
-            width: '100%'
-          }}
-        >
-          Add range
-        </Button>
-        <Button
-          bsStyle='danger'
-          className='pull-right'
+          className='exception-date-picker'
+          key={index}
           style={{
-            alignSelf: 'center',
-            gridColumn: '12',
-            height: '100%',
-            justifySelf: 'center',
-            padding: '2px 7.5px'
-          }}
-          key={`date-remove-${index}`}
-          onClick={this._onRemoveDate}>
-          <Icon type='times' />
-        </Button>
-        {isInvalid && date
-          ? <small style={{gridColumn: 'span 12'}}>
-            {moment(date).format('MM/DD/YYYY')} appears in another schedule exception. Please choose another date.
-          </small>
-          : null
-        }
-      </div>
+            alignItems: 'center',
+            display: 'grid',
+            gap: 5,
+            gridTemplateColumns: 'repeat(12, 1fr)',
+            justifyContent: 'center',
+            marginBottom: '5px',
+            position: 'relative',
+            textAlign: 'center'
+          }}>
+          <div style={{gridColumn: 'span 5'}}>
+            <DateTimeField
+              inputProps={{style: inputStyleProps}}
+              key={`date-${index}`}
+              mode='date'
+              {...dateTimeProps}
+            />
+          </div>
+
+          <Button
+            data-test-id='exception-add-range'
+            disabled={!date}
+            onClick={this._addRange}
+            style={{
+              gridColumn: '7 / 12',
+              height: '100%',
+              lineHeight: 0,
+              padding: '4'
+            }}
+          >
+            Add range
+          </Button>
+          <Button
+            bsStyle='danger'
+            className='pull-right'
+            style={{
+              alignSelf: 'center',
+              gridColumn: '12',
+              height: '100%',
+              justifySelf: 'center',
+              padding: '2px 7.5px'
+            }}
+            key={`date-remove-${index}`}
```

Comment:
Sort props.

Reply:
sorted in https://github.com/ibi-group/datatools-ui/pull/828/commits/bd59f94bd41bd062d40630c054cacfdc97f32a98